dependencies/softfloat/berkeley-softfloat-3/.clang-format

```diff
@@ -1,3 +0,0 @@
-# This is third-party code and should not be reformatted
-DisableFormat: true
-SortIncludes: Never
```

Comment:
Can we move this file to the top-level dependencies/ folder? That should make clang-format work as expected even when not run from a pre-commit hook.

The only reason I didn't do this before is that there wasn't a top-level dependencies folder.